plugins/storage/volume/linstor/src/main/java/com/cloud/hypervisor/kvm/storage/LinstorStorageAdaptor.java

```diff
@@ -302,26 +308,89 @@ public boolean connectPhysicalDisk(String volumePath, KVMStoragePool pool, Map<S
         return true;
     }
 
+    private Optional<ResourceWithVolumes> getResourceByPath(final List<ResourceWithVolumes> resources, String path) {
+        return resources.stream()
+                .filter(rsc -> rsc.getVolumes().stream()
+                        .anyMatch(v -> v.getDevicePath().equals(path)))
+                .findFirst();
+    }
+
+    private boolean tryDisconnectLinstor(String volumePath, KVMStoragePool pool)
+    {
+        s_logger.debug("Linstor: Using storpool: " + pool.getUuid());
```

Comment:
```suggestion
        s_logger.debug("Linstor: Using storage pool: " + pool.getUuid());
```

Reply:
we abbreviate storage pool as storpool in Linstor, but this could lead to confusion here :D